README.md

```diff
@@ -265,6 +265,61 @@ The static site that gets built is pushed to the `documentation` branch of this
 We host the site on netlify. On master branch builds (see `.github/workflows/documentation.yml`), we push the built docs to
 the `documentation` branch. Netlify automatically re-deploys the docs pages whenever there is a change to that branch.
 
+## Releases
+### Release Timeline
+**For Rasa Open Source, we usually commit to time-based releases, specifically on a monthly basis.** 
+This means that we commit beforehand to releasing a specific version of Rasa Open Source on a specific day, 
+and we cannot be 100% sure what will go in a release, because certain features may not be ready. 
+
+At the beginning of each quarter, the Rasa team will review the scheduled release dates for all products and make sure 
+they work for the projected work we have planned for the quarter, as well as work well across products.
+
+**Once the dates are settled upon, we update the respective milestones.**
+
+### Cutting a Major / Minor release
+#### A week before release day
+
+1. **Make sure the [milestone](https://github.com/RasaHQ/rasa/milestones) already exists and is scheduled for the 
+correct date**
```

Comment:
Well supposedly the milestone will already be there, as we discuss all releases in the beginning of each quarter. This step is mostly to make sure the information in the milestone is up to date. Also because 1) it shouldn't be the job of the person tasked with cutting the release to create the milestone and define the date of release and 2) it can be useful for the person tasked with cutting the release to take a quick look at the milestone anyway.

But anyway I can remove this if it adds confusion, no strong feelings about it 😊

Reply:
I fully agree with your points 👍 

For the 2.1 release we had to decide what should / can go in the release which in turn required the squads to plan for certain items in their sprint plannings. Considering that e.g. the loop squad has a biweekly sprint cycle, deciding on release content a week before the release doesn't seem to be timely enough. I'd vote for keeping this item, but changing it to `Two weeks. before the release day`. What do you think?

Reply:
Hmm, I'm not sure about this, because again these are instructions for whoever is cutting the release, and it should not be on their shoulders to sync with the squads on what *can* go into the release, that should be happening in parallel and between the squads and the product team. IMO, from the PoV of the person cutting the release, giving a first look at the milestone a week before the release day is more than enough to get an idea of what is still being worked on / what's in the release.